pkg/userdata/rhel/provider.go

```diff
@@ -194,13 +194,20 @@ write_files:
     yum install -y yum-utils
     yum-config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
 {{- /*	Due to DNF modules we have to do this on docker-ce repo
-		More info at: https://bugzilla.redhat.com/show_bug.cgi?id=1756473 */}}
+        More info at: https://bugzilla.redhat.com/show_bug.cgi?id=1756473 */}}
     sed -i 's/\$releasever/7/g' /etc/yum.repos.d/docker-ce.repo
     yum-config-manager --save --setopt=docker-ce-stable.module_hotfixes=true
 
     DOCKER_VERSION='{{ .DockerVersion }}'
-    yum install -y docker-ce-${DOCKER_VERSION} \
-      docker-ce-cli-${DOCKER_VERSION} \
+    yum install -y docker-ce-${DOCKER_VERSION} docker-ce-cli-${DOCKER_VERSION}
+
+    {{- if eq .CloudProviderName "azure" }}
```

Comment:
The current place where you add this line is kinda useless as the line number 194: `yum install -y yum-utils ` is already calling yum with the default rhui packages. Please move this block from this line to line number 193.

Reply:
thanks!